cEP-0000.md

```diff
@@ -29,8 +29,12 @@ Spelling
 As we all know, coala is written with a lower case c. The proper spelling of a
 cEP is `cEP-[0-9]{4}([a-z])?`, so `cEP-####` or `cEP-####x`.
 
-Invalid ways to express cEP specifications include `CEP-####`, `cEP####`,
-`cep-####`, and `cEP-#`.
+<NOTE> Leaving out the zeroes :-
+A cEP may also be spelled as `cEP-#` , `cEP-##` or `cEP-###` in case the
+author wants to.
```

Comment:
don't make this a note please, just add it to the previous sentence like it was always there.

Also don't forget to update the version of the cEP